package.json

```diff
@@ -20,14 +20,15 @@
     "test:coverage": "nyc report",
     "test:a11y": "yarn test:accessibility",
     "sonar-scan": "sonar-scanner",
-    "test:smoke": "NODE_ENV=test mocha test/smoke/**/*.test.ts --exit",
+    "test:smoke": "yarn codecept:e2e -g @smoke",
+    "test:smoke-local": "yarn codecept:e2e-local -g @smoke",
     "test:functional": "yarn codecept:e2e",
```

Comment:
shall we add a "functional" flag to "test:functional"? otherwise we would be running twice smoke test